src/sentry/static/sentry/app/components/events/interfaces/togglableAddress.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import Tooltip from 'app/components/tooltip';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import {formatAddress, parseAddress} from 'app/components/events/interfaces/utils';
+
+type Props = {
+  address: string;
+  startingAddress: string | null;
+  isAbsolute: boolean;
+  onToggle?: () => void;
+  isFoundByStackScanning: boolean;
+  isInlineFrame: boolean;
+  maxLengthOfRelativeAddress: number;
+};
+
+class TogglableAddress extends React.Component<Props> {
+  convertAbsoluteAddressToRelative = () => {
+    const {startingAddress, address, maxLengthOfRelativeAddress} = this.props;
+    if (!startingAddress) {
+      return '';
+    }
+
+    const relativeAddress = formatAddress(
+      parseAddress(address) - parseAddress(startingAddress),
+      maxLengthOfRelativeAddress
+    );
+
+    return `+${relativeAddress}`;
+  };
+
+  getAddressTooltip = () => {
```

Comment:
Does this need to be an arrow function? You aren't using it as an event handler.